.github/workflows/_local-promote-train.yaml

```diff
@@ -0,0 +1,71 @@
+name: Release Train
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: "Don't actually promote the charms, but print the promotions."
+        type: boolean
+        default: false
+        required: true
+    secrets:
+      CHARMHUB_TOKEN:
+        required: true
+
+jobs:
+  promote:
+    name: Promote Charm
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        charm-repo:
+          - alertmanager-k8s-operator
+          - avalanche-k8s-operator
+          - blackbox-exporter-k8s-operator
+          - catalogue-k8s-operator
+          # - cos-configuration-k8s-operator
+          # - cos-proxy-operator
+          # - grafana-agent-k8s-operator
+          # - grafana-agent-operator
+          # - grafana-k8s-operator
+          # - karma-alertmanager-proxy-k8s-operator
+          # - karma-k8s-operator
+          # - loki-k8s-operator
+          # - mimir-coordinator-k8s-operator
+          # - mimir-worker-k8s-operator
+          # - prometheus-k8s-operator
+          # - prometheus-pushgateway-k8s-operator
+          # - prometheus-scrape-config-k8s-operator
+          # - prometheus-scrape-target-k8s-operator
+          # - tempo-k8s-operator
+          # - traefik-k8s-operator
+          # - traefik-route-k8s-operator
```

Comment:
Committing commented out code?

Reply:
This PR only added the workflow file, so I could test it out; it'll be overwritten soon by a follow-up PR.